pypeit/pypeit.py

```diff
@@ -785,13 +789,19 @@ def objfind_one(self, frames, det, bg_frames, std_outfile=None):
                 slits=self.caliBrate.slits,  # For flexure correction
                 ignore_saturation=False), frame_par['process'])
 
+        # Deal with manual extraction
+        row = self.fitstbl[frames[0]]
+        manual_obj = ManualExtractionObj.by_fitstbl_input(
+            row['filename'], row['manual']) if len(row['manual'].strip()) > 0 else None
+
```

Comment:
You can simply convert this to the dict already here and then Reduce only depends on the dict but not the data container, which is preferred. See my comments below.